src/components/TextInputWithPrefix/index.android.js

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types';
+import {TextInput, View} from 'react-native';
+import _ from 'underscore';
+import React from 'react';
+import Text from '../Text';
+import styles from '../../styles/styles';
+
+const propTypes = {
+    /** Prefix character */
+    prefixCharacter: PropTypes.string,
+
+    /** Text to show if there is an error */
+    errorText: PropTypes.string,
+
+    /** Whether to disable the field and style */
+    disabled: PropTypes.bool,
+
+    /** Callback to execute the text input is modified */
+    onChangeText: PropTypes.func,
+};
+
+const defaultProps = {
+    errorText: '',
+    prefixCharacter: '',
+    disabled: false,
+    onChangeText: () => {},
+};
+
+const TextInputWithPrefix = props => (_.isEmpty(props.prefixCharacter)
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    ? <TextInput {..._.omit(props, ['prefixCharacter', 'errorText'])} />
```

Comment:
Same comment about props spreading as the one above.

Reply:
See my question in your other comment!

Reply:
Here, there are other `TextInput` props that are being passed implicitly in addition with the other props we've defined in propTypes, so I think it makes sense to use prop spreading here. Otherwise, I'd have to pass [every single one of these props](https://reactnative.dev/docs/textinput#props) individually, which seems unnecessary.